app/webpacker/components/RegistrationsV2/Register/RegistrationRequirements.jsx

```diff
@@ -0,0 +1,102 @@
+import { UiIcon } from '@thewca/wca-components';
```

Comment:
Addendum: I would like to avoid the use of `@thewca/wca-components` if its only purpose is the `UiIcon` thingy.
(In general, I am not too happy with the notion of `UiIcon` overall, because it ties `wca-components` to SemUI, which is not cool... But that's a whole different discussion to be had.)
Actionable item in terms of this review: Use SemUI `Icon` component instead and get rid of the `wca-components` dependency in this PR altogether.